test/hotspot/jtreg/serviceability/jvmti/vthread/VThreadEventTest/VThreadEventTest.java

```diff
@@ -136,6 +136,10 @@ private static void await(CountDownLatch dumpedLatch) {
     public static void main(String[] args) throws Exception {
         if (Runtime.getRuntime().availableProcessors() < 8) {
             log("WARNING: test expects at least 8 processors.");
+            if (System.getProperty("os.name").startsWith("Windows") && (Runtime.getRuntime().availableProcessors() < 16)) {
+                // On Windows platform, ensure sufficient processor available to avoid deadlock.
+                return;
```

Comment:
I'm thinking we should print a message that there aren't enough processors and the test isn't running, to help avoid any confusion down the line.

Reply:
I've added new warning message to clarify this. Will update the other porting PRs.